cobbler/package/clean.sls

```diff
@@ -9,6 +9,12 @@
 include:
   - {{ sls_config_clean }}
 
+{% if pkg.communityrepo.enabled %}
+cobbler-package-clean-repository-removed:
+  pkgrepo.absent:
+    - name: cobbler_community_repo
+{% endif %}
```

Comment:
I might be overlooking something, but if `pkg.communityrepo` is _enabled_, should it really be removed? It seems to conflict with it being installed in `repository.sls`.

Reply:
As far as the upstream methodology goes it is desired to have a `<formula>.clean` state that is responsible for uninstallation of the formula. As such I believe it should not be a conflict. The "substate" `clean` should be present in other areas as well.